mm/kfence.c

```diff
@@ -733,6 +735,7 @@ void kfence_observe_memcg_cache(struct kmem_cache *memcg_cache)
 	unsigned long flags;
 	struct kmem_cache *root;
 	char *name;
+	int cpu = smp_processor_id();
```

Comment:
If some random CPU, where the current one is the most likely one, is what you want, need to use raw_smp_processor_id().

Otherwise move it inside the IRQ-off region where it's used.

Reply:
Thanks!